README.md

```diff
@@ -25,6 +25,17 @@ everything is working properly. Output, checkpoint and formatted checkpoint
 files will be generated in the output directory. A log showing the steps
 performed by the script will be available in the log directory.
 
+### Running on different nodes
+
+If your job needs more time, you can run on a different node. The default node is 'short' 
+which has a 24 h time limit. Switching to 'long' changes the time limit to 2 weeks. A full 
+list of nodes can be found [here](https://hpcrcf.atlassian.net/wiki/spaces/TCP/pages/7285967/Partition+List).
```

Comment:
This information seems to be specific to UOregon, don't you have a fork of this repo specifically for your lab and UOregon use cases? or would you rather just leave it here?

Reply:
I'd rather just use this as the lab specific one.